Android/Android-Unreal-Plugin/src/main/java/com/applovin/unreal/AppLovinMAX.java

```diff
@@ -0,0 +1,1326 @@
+package com.applovin.unreal;
+
+import android.app.Activity;
+import android.content.Context;
+import android.content.pm.ApplicationInfo;
+import android.content.pm.PackageManager;
+import android.graphics.Color;
+import android.graphics.Rect;
+import android.os.Bundle;
+import android.text.TextUtils;
+import android.util.Log;
+import android.view.Gravity;
+import android.view.OrientationEventListener;
+import android.view.View;
+import android.view.ViewGroup;
+import android.view.ViewParent;
+import android.widget.LinearLayout;
+import android.widget.RelativeLayout;
+
+import com.applovin.mediation.MaxAd;
+import com.applovin.mediation.MaxAdFormat;
+import com.applovin.mediation.MaxAdListener;
+import com.applovin.mediation.MaxAdViewAdListener;
+import com.applovin.mediation.MaxError;
+import com.applovin.mediation.MaxReward;
+import com.applovin.mediation.MaxRewardedAdListener;
+import com.applovin.mediation.ads.MaxAdView;
+import com.applovin.mediation.ads.MaxInterstitialAd;
+import com.applovin.mediation.ads.MaxRewardedAd;
+import com.applovin.sdk.AppLovinMediationProvider;
+import com.applovin.sdk.AppLovinPrivacySettings;
+import com.applovin.sdk.AppLovinSdk;
+import com.applovin.sdk.AppLovinSdkConfiguration;
+import com.applovin.sdk.AppLovinSdkSettings;
+import com.applovin.sdk.AppLovinSdkUtils;
+
+import java.lang.ref.WeakReference;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+/**
+ * Created by Ritam Sarmah on May 5 2021
+ */
+public class AppLovinMAX
```

Comment:
I'd keep this naming consistent with the iOS equivalent, `MAUnrealPlugin`. It seems you took a cue from ReactNative here and Unity for iOS.

Reply:
Actually I wonder if I should consider renaming the iOS side, since it seems all of our other "newer" plugins use "AppLovinMAX" for iOS and Android?

Reply:
Never mind I forgot that the "AppLovinMAX" header file name would collide with the name on the Unreal side. I'll rename Android